libs/application/templates/party-letter/src/lib/PartyLetterApplicationTemplate.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  ApplicationTemplate,
+  ApplicationTypes,
+  ApplicationContext,
+  ApplicationRole,
+  ApplicationStateSchema,
+  Application,
+  DefaultEvents,
+} from '@island.is/application/core'
+import * as z from 'zod'
+
+const nationalIdRegex = /([0-9]){6}-?([0-9]){4}/ // TODO: make this use kennitala lib
```

Comment:
You might want to do it already, since we already have it in the project for example [here](https://github.com/island-is/island.is/blob/main/libs/application/templates/parental-leave/src/lib/dataSchema.ts#L63)

Reply:
True, fixed